### DISCUSSION TITLE
Would loading a script tag of compiled java code (as JS) work?

### DISCUSSION THREAD

**Discussion by STPv22** [eaglerforge/EaglerForge#37] [Ideas]
I know very little of the subject

**Comment by leahanderson1** [comment]
well yeah i guess if you used teavm and allat

**Comment by STPv22** [comment]
ye

**Comment by STPv22** [comment]
Ill test soon. Give me sum code that will work and ill set it up myself?

**Reply by leahanderson1** [reply]
just make a simple hello world class

**Comment by STPv22** [comment]
alr

**Comment by STPv22** [comment]
kinda struggling to set up

**Comment by radmanplays** [comment]
it is possible but you need to change the compile options (gradle.build file) to put the java code in a javascript function WITHOUT the name "main" and also it is impossible to directly access the modAPI unless you use teavm's libraries to access javascript from java and make modAPI's methods accessable from java.

**Reply by STPv22** [reply]
Noted thx

**Reply by STPv22** [reply]
How tf do i do that

**Comment by STPv22** [comment]
I managed to get it to work with `alert(msg);` but it was worth a shot
![Screenshot 2024-09-08 2 12 10 AM](https://github.com/user-attachments/assets/6d974755-b534-4607-84ec-9504d45c066a)
![Screenshot 2024-09-08 2 12 01 AM](https://github.com/user-attachments/assets/2a0e8342-257e-4939-bf21-e3994d0ca771)
![Screenshot 2024-09-08 2 11 47 AM](https://github.com/user-attachments/assets/8cba32c6-c09b-4214-8ecb-124eae5673cf)

**Reply by STPv22** [reply]
its 2:15 AM rn lmao
